images/base/files/usr/local/bin/entrypoint

```diff
@@ -435,14 +464,25 @@ enable_network_magic(){
       if [[ -z $curr_ipv6 ]]; then
         echo "ERROR: Have an old IPv6 address but no current IPv6 address (!)" >&2
       fi
-      # kubernetes manifests are only present on control-plane nodes
-      sed -i "s#${old_ipv6}#${curr_ipv6}#" /etc/kubernetes/manifests/*.yaml || true
-      # this is no longer required with autodiscovery
-      sed -i "s#${old_ipv6}#${curr_ipv6}#" /var/lib/kubelet/kubeadm-flags.env || true
+      if [[ "${old_ipv6}" != "${curr_ipv6}" ]]; then
+        sed_ipv6_command="s#\b$(regex_escape_ip "${old_ipv6}")\b#${curr_ipv6}#g"
+        # kubernetes manifests are only present on control-plane nodes
+        sed -i "${sed_ipv6_command}" /etc/kubernetes/manifests/*.yaml || true
```

Comment:
we can loop over these as well instead, just add it to the loop below?

Reply:
er no, if we put the wildcard in the list of for loop items it will expand inline as individual loop items

Reply:
(and if nothing matches the wildcard there will be no additional loop items)

Reply:
Thanks for the tips. It seems when any file matches, it's expanded as expected, but it's not expanded when nothing matches, it will check file existence with wildcard, though the check will return false which will prevent further operation, do you think this is expected? is it legal to check file existence with wildcard?
```
# for f in /etc/emptydir/*.yaml /etc/kubernetes/controller-manager.conf; do echo "testing $f"; done
testing /etc/emptydir/*.yaml
testing /etc/kubernetes/controller-manager.conf
```

Reply:
🤯 I totally forgot that no match => the wildcard pattern itself 🤯 

let's just keep the explicit file list + check for now, thanks!